docroot/themes/contrib/civictheme/civictheme_library/components/00-base/typography/typography.stories.js

```diff
@@ -1,22 +1,230 @@
 
   for (const i in Object.values(typographyNames)) {
-    html += `<div class="example-container">`;
-    html += `<div class="example-container__title">${typographyNames[i].charAt(0).toUpperCase()}${typographyNames[i].replace('-', ' ').slice(1)} <code>${typographyNames[i]}</code></div>`;
-    html += `<div class="example-container__content civictheme-${typographyNames[i]}">The quick brown fox jumps over the lazy dog</div>`;
-    html += `</div>`;
+    let content = randomSentence(50);
+    let title = capitalizeFirstLetter(typographyNames[i].replace('-', ' '));
+    title += `<code>${typographyNames[i]}</code>`;
+
+    if (typographyNames[i].split('-')[0] === 'heading') {
+      content = CivicThemeHeading({
+        theme: () => theme,
+        content: 'The quick brown fox jumps over the lazy dog',
+        level: htag++,
+      });
+    }
+
+    if (typographyNames[i].split('-')[0] === 'label') {
+      content = randomText(2);
+    }
+
+    html += ExampleContainer(title, content, typographyNames[i]);
   }
 
-  return html;
+  let paragraph = `<p>${randomSentence(250)}</p>`;
+  html += ExampleContainer('Paragraph 1', paragraph, 'content');
+
+  paragraph = `
+    <p>Pellentesque in ipsum id orci porta dapibus. ${ExampleLink(theme, 'Internal link', '/test1', false)} convallis at tellus Curabitur non nulla sit amet nisl tempus convallis quis ac lectus.
+    Pellentesque in ipsum id orci porta dapibus ${ExampleLink(theme, 'Internal link 2', '/test12', false)}.
+    </p>
+    <p>Quisque velit nisi, pretium ut lacinia in, elementum id enim. ${ExampleLink(theme, 'External link', randomUrl(), true)} Donec rutrum congue leo eget malesuada.
+    Cras ultricies ligula sed magna dictum porta. ${ExampleLink(theme, 'External link 2', randomUrl(), true)}
+    Praesent sapien massa, convallis a pellentesque nec, egestas non nisi.</p>
+
+    <p>Internal links:
+      <ul>
+        <li>${ExampleLink(theme, 'Internal link example 1', '/test1', false)}</li>
+        <li>${ExampleLink(theme, 'Internal link example 2', '/test12', false)}</li>
+      </ul>
+    </p>
+    <p>External links:
+      <ul>
+        <li>${ExampleLink(theme, 'External link example 1', randomUrl(), true)}</li>
+        <li>${ExampleLink(theme, 'External link example 2', randomUrl(), true)}</li>
+      </ul>
+    </p>
+
+    <p><strong>Bold: Nulla quis lorem ut libero malesuada feugiat.</strong></p>
+    <p><em>Italic: Nulla quis lorem ut libero malesuada feugiat.</em></p>
+    <p><span style="text-decoration: underline;">Underline: Nulla quis lorem ut libero malesuada feugiat.</span></p>
+    <p><span style="text-decoration: line-through;">Strikethrouh: Nulla quis lorem ut libero malesuada feugiat.</span></p>
+    <p>Date: 20<sup>th&nbsp;</sup>August 2022</p>
+    <p>H<sub>2</sub>0</p></p>
+  `;
+  html += ExampleContainer('Paragraph 2', paragraph, 'content');
+
+  paragraph = `
+    <p>${randomSentence(50)}</p>
+    <div class="example-container__title">Unordered list</div>
+    <ul>
```

Comment:
can you please create a generator for these

Reply:
Generator for UL/LI ?

Reply:
yes. we have a generator for Lorem/Ipsum. can you please create a generator for UL/LI

Reply:
actually - twig template is enough. please ignore my comment above